test/control/manifest.json

```diff
@@ -0,0 +1,75 @@
+{
+    "http_server": {
+        "addr": ":9999",
+        "dir": "../_res",
+        "testmode": false
+    },
+    "name": "check control structures in array",
+    "tests": [
+        {
+            "name": "check control structures in array (MUST FAIL)",
+            "request": {
+                "server_url": "http://localhost:9999",
+                "endpoint": "bounce-json",
+                "method": "POST",
+                "body": {
+                    "_files": [
+                        {
+                            "path": "Test XML+CSV+JSON/1-2.csv"
+                        },
+                        {
+                            "path": "Test XML+CSV+JSON/1-2.json"
+                        },
+                        {
+                            "path": "files/no-1-sid-10-berlin.jpg"
+                        },
+                        {
+                            "path": "files/no-2-sid-10-mona-lisa-1.jpg"
+                        },
+                        {
+                            "path": "files/no-3-sid-9-Gustav_Klimt_016.jpg"
+                        },
+                        {
+                            "path": "files/no-4-sid-9-art-exposition-735518_1920_big_small.png"
+                        }
+                    ]
+                }
+            },
+            "response": {
+                "statuscode": 200,
+                "body": {
+                    "body": {
+                        "_files": [
+                            {
+                                "path:control": {
+                                    "match": "^.*Gustav_Klimt_016.*$"
+                                }
+                            },
+                            {
+                                "path:control": {
+                                    "match": "^.*art-exposition-735518_1920.*$"
+                                }
+                            },
+                            {
+                                "path:control": {
+                                    "match": "^.*berlin.*$"
+                                }
+                            },
+                            {
+                                "path:control": {
+                                    "match": "^.*mona-lisa-1.*$"
+                                }
+                            },
+                            {
+                                // XXX this control structure needs to cause a test failure: there is no path that matches "^.*henk.*$"
+                                "path:control": {
+                                    "match": "^.*henk.*$"
+                                }
+                            }
```

Comment:
Can we test the problem & fix with a test which does not require to fail? The apitest apitests should run without failure. We might need a different approach to test the failing apitest with and "OK"